website/vercel.json

```diff
@@ -747,116 +747,124 @@
       "destination": "https://hub.cloudquery.io/plugins/destination/cloudquery/sqlite"
     },
     {
-      "source": "/integrations/file",
-      "destination": "https://hub.cloudquery.io/plugins/source/cloudquery/file"
-    },
-    {
-      "source": "/integrations/salesforce",
-      "destination": "https://hub.cloudquery.io/plugins/source/cloudquery/salesforce"
-    },
-    {
-      "source": "/integrations/azuredevops",
+      "source": "/integrations/azuredevops/:path*",
       "destination": "https://hub.cloudquery.io/plugins/source/cloudquery/azuredevops"
     },
     {
-      "source": "/integrations/facebookmarketing",
+      "source": "/integrations/facebookmarketing/:path*",
       "destination": "https://hub.cloudquery.io/plugins/source/cloudquery/facebookmarketing"
     },
     {
-      "source": "/integrations/fastly",
+      "source": "/integrations/fastly/:path*",
       "destination": "https://hub.cloudquery.io/plugins/source/cloudquery/fastly"
     },
     {
-      "source": "/integrations/googleanalytics",
-      "destination": "https://hub.cloudquery.io/plugins/source/cloudquery/googleanalytics"
-    },
-    {
-      "source": "/integrations/homebrew",
-      "destination": "https://hub.cloudquery.io/plugins/source/cloudquery/homebrew"
-    },
-    {
-      "source": "/integrations/shopify",
-      "destination": "https://hub.cloudquery.io/plugins/source/cloudquery/shopify"
-    },
-    {
-      "source": "/integrations/snyk",
-      "destination": "https://hub.cloudquery.io/plugins/source/cloudquery/snyk"
-    },
-    {
-      "source": "/integrations/stripe",
-      "destination": "https://hub.cloudquery.io/plugins/source/cloudquery/stripe"
+      "source": "/integrations/file/:path*",
+      "destination": "https://hub.cloudquery.io/plugins/source/cloudquery/file"
     },
     {
-      "source": "/integrations/terraform",
-      "destination": "https://hub.cloudquery.io/plugins/source/cloudquery/terraform"
+      "source": "/integrations/gandi/:path*",
```

Comment:
I don't think any other paths ever existed under the premium plugins, but it probably doesn't matter